Makefile

```diff
@@ -50,7 +50,12 @@ docker: ginkgo
 ifeq ($(RUNTIME),)
 	$(error RUNTIME is not set)
 else
-	./ginkgo -v -focus "${FOCUS}" -skip "${SKIP}" ./integration/docker/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
+	# Run tests in parallel, skip tests that need to be run serialized
+	./ginkgo -p -v -focus "${FOCUS}" -skip "${SKIP}" -skip "Serial Test" \
+		./integration/docker/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
+	# Now run serialized tests
+	./ginkgo -v -focus "${FOCUS}" -focus "Serial Test" -skip "${SKIP}" \
```

Comment:
`-focus "[Serial Test]"` to be more specific

Reply:
Already tried it, but seems like using `[` `]` didn't work. Let me check if escaping them.

Reply:
yes, I think you should uses `\[` ans `\]`

Reply:
@devimc fixed.